### DISCUSSION TITLE
Optional parameter $force declared before required (After Update)

### DISCUSSION THREAD

**Discussion by spooksie** [livewire/livewire#7330] [7. Bugs (archived)] [answered]
### Livewire version

3.2.0

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

After composer update, I'm getting this error in Ray. It keeps on appearing.

`Optional parameter $force declared before required parameter $testType is implicitly treated as a required parameter in /Users/bit/Herd/podswap/vendor/livewire/livewire/src/Features/SupportConsoleCommands/Commands/MakeCommand.php on line 111`

HandleExceptions.php:105

![Screenshot 2023-11-20 at 5 44 00 PM](https://github.com/livewire/livewire/assets/220888/ce2c58de-1cae-4758-9371-2ab12dbe2029)

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment, marked as answer]
@spooksie This has already been fixed and released in v3.2.1 🙂
